packages/url_launcher/url_launcher_platform_interface/test/link_test.dart

```diff
@@ -83,6 +83,9 @@ class _RouteDelegate extends RouterDelegate<RouteInformation>
     if (_history.isEmpty) {
       return const Placeholder(key: Key('empty'));
     }
+    // TODO(chunhtai): remove the ignore once migration completes.
+    // https://github.com/flutter/flutter/issues/100624.
+    // ignore: unnecessary_string_interpolations, deprecated_member_use
```

Comment:
Why is unnecessary_string_interpolations here?

Reply:
The change also makes location non nullable